frontend/src/lib/k8s/cluster.ts

```diff
@@ -243,6 +244,18 @@ export function makeKubeObject<T extends KubeObjectInterface | KubeEvent>(
       return this.apiEndpoint.put(data);
     }
 
+    patch(body: OpPatch[]) {
```

Comment:
I don't understand what this code does, or why it is different to delete and put with regards to args handling?

Reply:
Started some docs for patch()...

There's a few things that could be improved in my docs:
- I had trouble finding what this method returns because it's being typed as any.
- Added a link to OpPatch docs I found, because the docs for json-patch aren't there and the typing means you need to read the node-modules source code.
- wasn't sure of a realistic example.

```ts
    /**
     * Patches the cluster with a set of operations.
     * 
     * @param body an array of patch operations
     * @returns ? TODO: type is being resolved as any. The apiFactory.patch needs typing.
     * 
     * @see {@link https://www.npmjs.com/package/json-patch#patch-operations} OpPatch docs
     * @example
     * patch([{op: 'add', path: '/foo', value: 'bar'}]) // what's a more realistic example?
     */
    patch(...)
```

Reply:
I think I'd like to continue documenting and such in this PR: https://github.com/kinvolk/headlamp/issues/465